doc/sphinx/source/project/introduction.rst

```diff
@@ -4,176 +4,86 @@
 
-.. figure:: ../graphics/dl-current-state.png
-   :width: 700px
-   :alt: Scalability matters
-
-   Figure F: The number of kernels necessary to achieve optimal performance is 
-   bounded by the product of the number of chip designs one wishes to support, 
-   the number of data types supported, the number of operations, and the 
-   cardinality of each parameter for each operation.
-
-In the past, this upper bound was quite limited; however, since the industry is 
-shifting toward a more diverse future in terms of deep learning hardware, the 
-number of distinct kernels is exploding and will continue to explode.
-
-
-Get the best of both worlds
----------------------------
+3. Increasing number of kernels 
+---------------------------------------------------------
 
-Integrating a framework on nGraph can be an attractive option for hardware 
-companies trying to design their own deep learning hardware or network architecture. 
-Framework integration is non-trivial amount of work, and nGraph automatically 
-does much of the heavy lifting. Furthermore, PlaidML can provide a wide range of 
-hardware coverage and optimization automatically. Any hardware that supports 
-LLVM, OpenCL, OpenGL, CUDA or Metal can be supported automatically with PlaidML 
-and nGraph.  
+As mentioned in the pervious section, kernel libraries need to be integrated with multiple deep learning frameworks, and this arduous task becomes even harder due to increased numbers of required kernels for achieving optimial performance. The number of required kernels is product of number of chip designs, data types, operations, and the cardinality of each parameter for each operation. In the past, the number of required kernels was limited, but as the AI research and industry rapidly develops, the final product of required kernels is increasing exponentially. 
 
-.. _figure-G:
+.. _figure-C:
 
-.. figure:: ../graphics/graph-compilers-at-a-glance.png
-   :width: 700px
-   :alt: Overview of various graph and tensor compilers.
+.. figure:: ../graphics/intro_kernel_explosion.png
+   :width: 555px
+   :alt: 
 
-   Figure G: Overview of various graph and tensor compilers.
+PlaidML was designed to address the expoential growth of kernel needs. It takes two inputs: operation defined by the user and machine description of the targetted hardware. It utilizes a Domain Specific Language (DSL) called Tile which allows developers to express how an operation should calculate tensors in a intutitive mathematical form. PlaidML takes user defined Tile code along with targed machine description such as threads, max memory input, etc to automatically apply low level optimizations. This automated optimization does not require kernel libraries to be written and lifts heavy burden for kernel developers. It also provides flexibility to support newer deep learning models in absence of hand optimized kernels for the new operations.   
```

Comment:
PlaidML addresses the kernel explosion problem in a manner that lifts a heavy burden off kernel developers.  It automatically lowers networks from nGraph into Tile, a :abbr:`Domain-Specific Language (DSL)` designed for deep learning that allows developers to express how an operation should calculate tensors in a intuitive mathematical form.  Integration of PlaidML with nGraph means extra flexibility to support newer deep learning models in absence of hand optimized kernels for the new operations.